README.md

```diff
@@ -37,28 +37,11 @@ The following files and folders enable our continuous integration system.
 
 ## Installation
 
-1. Make sure you have the appropriate ROS version installed for your version of Ubuntu.
+This repository should be cloned into the src directory of a catkin workspace. Use ```catkin_make``` in the workspace directory to build the code. (NOTE: Be sure to ```source devel/setup.sh``` before referencing roboracing packages.)
 
-2. Clone this repo into the _src_ directory of your catkin workspace.
-
-3. Also clone https://github.com/intel-ros/realsense and https://github.com/ros-drivers/pointgrey_camera_driver into the same _src_ directory
-
-4. Install the remaining dependencies with the following command in your catkin workspace folder:
-
-   ```sh
-   rosdep install --from-path src --ignore-src -y
-   ```
-
-5. Install python dependencies (for Python 2) which are not in ROS:
-
-   ```
-   pip install keras tensorflow numpy pynput matplotlib
-   ```
-
-6. Install librealsense following the instructions at https://github.com/IntelRealSense/librealsense/blob/master/doc/distribution_linux.md
-
-7. You should now be able to build the project by running `catkin_make` in the catkin_ws directory.
+For a guide on installing our code please go to [our guide](https://wiki.robojackets.org/RoboRacing_Software_Installation_Instructions).
 
+Join our [Slack](https://robojackets.slack.com/)!
```

Comment:
I do not think this is the appropriate spot for this to be. It adds some kind of idea that if someone stumbled on our code, they could join and we might support it (which according to our MIT license we do not). New members should join the slack because we pester them at meetings to.